CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ add_subdirectory(Dependencies/SafeDisc)
 add_subdirectory(Dependencies/Utility)
 add_subdirectory(resources)
 
+add_subdirectory(Common)
```

Comment:
I do wonder, should we perhaps name this "Base" instead of "Common"? There are other "Common" folders in the folder tree of the games, so perhaps it is better to use a different name for this reason, to avoid potential ambiguities with a `#include "Common/File.h"`?